interfaces/builtin/alsa.go

```diff
@@ -45,6 +45,19 @@ const alsaConnectedPlugAppArmor = `
 @{PROC}/asound/** rw,
 `
 
+const alsaConnectedPlugUdev = `
+# This file contains udev rules for alsa devices.
+#
+# Do not edit this file, it will be overwritten on updates
+
+KERNEL=="controlC[0-9]*", NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="hw[CD0-9]*",     NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="pcm[CD0-9cp]*",  NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="midiC[D0-9]*",   NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="timer",          NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="seq",            NAME="snd/%%k", TAG+="%[1]s"
```

Comment:
Okay, got your point.
Let's remove NAME key for each rule of the interfaces.
I'll also give it a try on snap to check if it works for me with the fine-tuned udev rules.
Thanks.